### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Oghenebrume50**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      render 'new'
+    end
+
+  end
+
+  def show
+    @event = Event.find(params[:id])
+  end
+
+  def attend
```
Comment (zlayabekrija) [inline app/controllers/events_controller.rb]: Attend method is part of EventAttandace model, please remove it from events controller and place it in the appropriate controller.

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  def new
+  end
+
+  def create
+    user = User.find_by(username: params[:session][:username])
+    if user
+      log_in user
+      redirect_to user_path(user.id)        
+    else
+      # flash.now[:danger] = 'Invalid email/password combination'
```
Comment (zlayabekrija) [inline app/controllers/sessions_controller.rb]: Uncomment this flash error it should let the user know if he did something wrong.

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  def show
+    @upcoming_events = current_user.upcoming_events
+    @prev_events = current_user.previous_events
+  end
+
+  def new
+  end
+
+  def create
```
Comment (zlayabekrija) [inline app/controllers/users_controller.rb]: This part can't be reached add this moment, please fix it.

**Comment by Oghenebrume50** [comment]
@rvvergara 
We have editted the codes, and users can actually attend events from the events index page

This screenshot shows the user's page with no attended events except created events
UserProfile

This shows all Events  seperated into past and upcoming, the past events has no attend button because one cannot attend past events, the upcoming has attend button by the side, except for those already attended, see next screenshot
AllEvents

This shows when the attend button has been clicked by a user, for the first two events under the upcoming events
AttendEvent

This shows that the clicked events appear on the user's show page properly
UserProfile2

**Comment by rvvergara** [comment]
Hi @Oghenebrume50 and @truetechcode ,

Great job!! Now the project has satisfied all minimum requirements per our checklist. You may now merge with `master` and submit a project completion form.

# :+1: 

# :champagne: :champagne:

**Pull request merged**